### PR TITLE
feat: add HNU careers feed

### PR DESCRIPTION
**Pull request by ningyougan**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/hnu/careers
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

湖南大学校园招聘

### CODE REVIEW

File: lib/v2/hnu/careers.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const currentUrl = `${rootUrl}module/getcareers?start_page=1&type=inner&day=&count=20&start=1`;
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+    const company_list = response.data.data;
+
+    ctx.state.data = {
+        title: `湖南大学 校内招聘`,
+        link: currentUrl,
```
Comment (TonyRL) [inline lib/v2/hnu/careers.js:14]: `link` should link to a human-readable website instead of API response.

Comment (TonyRL) [reply lib/v2/hnu/careers.js:17]: Please take a look at the `link` definition in the [documentation](https://docs.rsshub.app/en/joinus/advanced-feed.html).

> The URL of the website associated with the feed, which should link to a **human-readable** website

In the [Outputting the RSS](https://docs.rsshub.app/en/joinus/new-rss/start-code.html#via-api-outputting-the-rss) example, even though we obtain the data through an API (the issue list), `ctx.state.data.link` still provides a link to a human-readable website that serves the API responses.

Comment (ningyougan) [reply lib/v2/hnu/careers.js:17]: I know that. but i hope i can show the important information at the feed. so I add more info in the item. if I only show the info which is in the docs, i think this feed isn't good RSS what i want to see.

If I must comply with the document rules, then I can modify it as needed.

File: lib/v2/hnu/careers.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    // 其中有用的参数为count，为获取招聘信息的数量。
+    const currentUrl = `${rootUrl}module/getcareers?start_page=1&type=inner&day=&count=20&start=1`;
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+    const company_list = response.data.data;
+
+    ctx.state.data = {
+        title: `湖南大学 校内招聘`,
```
Comment (TonyRL) [inline lib/v2/hnu/careers.js:16]:
```suggestion
        title: '湖南大学 校内招聘',
```

File: lib/v2/hnu/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'hnu.edu.cn': {
+        _name: '湖南大学',
+        scc: [
+            {
+                title: '湖南大学 校内招聘',
```
Comment (TonyRL) [inline lib/v2/hnu/radar.js:6]:
```suggestion
                title: '校内招聘',
```

File: lib/v2/hnu/careers.js (hunk 1 above)
Comment (TonyRL) [reply lib/v2/hnu/careers.js:17]: Please look at the [example](https://docs.rsshub.app/en/joinus/new-rss/start-code.html#via-api-outputting-the-rss) carefully.
You don't present the users with the [API response](https://api.github.com/repos/DIYgod/RSSHub/issues) when he/she uses the open related website of an RSS feed from their RSS reader which is basically a big pile of text. You present a human-readable [site](https://github.com/DIYgod/RSSHub/issues).

File: lib/v2/hnu/radar.js (hunk 3 above)
Comment (ningyougan) [reply lib/v2/hnu/radar.js:6]: I'm sorry, I just understood what you meant. This was my mistake due to carelessness. Link should not be an API interface, and thank you very much for pointing out this bug. It has been fixed. Thank you.

**Pull request merged**